web/src/kitchensink/views/TupleView.vue

```diff
@@ -0,0 +1,32 @@
+<script lang="ts" setup>
+import TupleInput from "@/components/TupleInput.vue"
+import type { InputField } from "@/domain"
+import { defineDateField } from "@/domain"
+
+const tupleFields: { parent: InputField; child: InputField } = {
+  parent: defineDateField(
+    "decisionDate",
+    "Entscheidungsdatum",
```

Comment:
Why is this a concrete field name here? I would have expected `TupleView.vue` to be abstract since we do the definitions of fields usually in `coreDataFields.ts`?

Reply:
(kitchensink)